src/components/Button/Button.scss

```diff
@@ -0,0 +1,67 @@
+.uir-Button {
+    position: relative;
+    border: 0; /* override browser's default */
+    color: rgba(47,40,51,.6);
+    min-width: 50px;
+    height: 36px;
+    padding: 9px 18px;
+    margin: 8px 8px 8px 0;   
+    border-radius: 2px;
+    text-align: center;
+    white-space: nowrap;
+    text-transform: uppercase;
+    letter-spacing: 1px;
+    transition: all .3s;
+    font-family: "DINOT", sans-serif;
+    box-sizing: border-box;
+    font-size: 12px;
+    vertical-align: middle;
+    user-select: none;
+    outline: none;
+    background-color: rgba(47, 40, 51, .05);
+}
+
+.uir-Button-clear {
```

Comment:
If we're going to use SASS, we should take advantage of its nice features, like rule nesting:

```sass
.uir-Button {
	...
	&-clear {
		...
	}
	&:hover {
		...
	}
}
```

Reply:
Agree, I converted css to sass file without using any sass powers

These are variables I created for previous m-dash-ui (top 40 lines)
https://github.com/Mudano/m-dash-ui/blob/master/css/vars.scss 
The fact that designers work with semi-transparent backgrounds (maybe in the near future maybe even blur backgrounds), makes it a bit more difficult to calculate the correct backgrounds for all the variations. I used sass `rgba` functions to apply opacity to hex colors

Color variables need to get approved by Antoine / Makis